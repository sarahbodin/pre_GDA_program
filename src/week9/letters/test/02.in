7
aaabbb
bbbaaa
bababa
ababab
baaabb
abbbaa
abbbba
