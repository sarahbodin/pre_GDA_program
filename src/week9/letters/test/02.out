Case #1: a
Case #2: b
Case #3: b
Case #4: a
Case #5: b
Case #6: a
Case #7: b
